charts/policies/user-group-psp/questions.yaml

```diff
@@ -0,0 +1,158 @@
+# This is a Rancher questions file
+---
+questions:
+# General policy questions:
+- variable: "metadata.name"
+  type: string
+  default: user-group-psp
+  required: true
+  description: Name of policy resource
+  group: Settings
+  label: Name of policy resource
+- variable: "kind"
+  type: enum
+  options:
+    - "ClusterAdmissionPolicy"
+    - "AdmissionPolicy"
+  default: "ClusterAdmissionPolicy"
+  required: true
+  label: kind of policy resource
+  description: Either a cluster-wide policy (ClusterAdmissionPolicy) or a namespaced one (AdmissionPolicy)
+  group: Settings
+  show_subquestion_if: "AdmissionPolicy"
+  subquestions:
+    - variable: "metadata.namespace"
+      type: string
+      default: "default"
+      required: true
+      label: Namespace of the AdmissionPolicy
+      description: Namespace that the AdmissionPolicy will be deployed into and apply for.
+- variable: "spec.mode"
+  type: enum
+  options:
+    - "monitor"
+    - "protect"
+  default: "protect"
+  required: true
+  label: Execution mode of the policy
+  description: Execution mode of the policy. "protect" will reject requests,
+    "monitor" will log them. Transitioning from "monitor" to "protect" is
+    allowed, but not from "protect" to "monitor".
+  group: Settings
+- variable: "spec.mutating"
+  type: boolean
+  default: false
+  required: true
+  label: mutating
+  description: Whether the policy has the ability to mutate incoming requests or
+    not.
+  group: Settings
+
+# Specific policy settings questions:
+-
+  variable: "spec.settings.run_as_user.rule"
```

Comment:
This is the `user-group-psp` policy. Contains several enums, and if one selects `MustRunAs`, one gets an `overwrite` question entry.

It would be needed to change `mutating` to `true` if `overwrite` is also `true`, but that isn't possible with the current `questions.yaml` functionality.

[user-group-psp.webm](https://user-images.githubusercontent.com/2196685/188898601-76785d9e-eb27-43f7-ab81-f6a9f0683169.webm)